File: Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/IndexHasChanged.cs
using Raven.Client.Documents;
using Raven.Client.Documents.Indexes;
using Raven.Client.Documents.Operations.Indexes;

namespace Raven.Documentation.Samples.ClientApi.Operations
{

    public class IndexHasChagned
    {
        private interface IFoo
        {
            /*
            #region index_has_changed_1
            public IndexHasChangedOperation(IndexDefinition definition)
            #endregion
            */
        }

        public IndexHasChagned()
        {

            using (var store = new DocumentStore())
            {
                IndexDefinition ordersIndexDefinition;
                #region index_has_changed_2
                bool ordersIndexHasChanged = store.Maintenance.Send(new IndexHasChangedOperation(ordersIndexDefinition));
                #endregion
            }
        }
    }
}

File: Documentation/5.2/Samples/java/src/test/java/net/ravendb/ClientApi/Operations/Maintenance/Indexes/IndexHasChanged.java
package net.ravendb.ClientApi.Operations;

import net.ravendb.client.documents.DocumentStore;
import net.ravendb.client.documents.IDocumentStore;
import net.ravendb.client.documents.indexes.IndexDefinition;
import net.ravendb.client.documents.operations.indexes.IndexHasChangedOperation;

public class IndexHasChanged {


    private interface IFoo {
        /*
        //region index_has_changed_1
        public IndexHasChangedOperation(IndexDefinition definition)
        //endregion
        */
    }

    public IndexHasChanged() {
        try (IDocumentStore store = new DocumentStore()) {
            IndexDefinition ordersIndexDefinition = null;
            //region index_has_changed_2
            Boolean ordersIndexHasChanged =
                store.maintenance().send(new IndexHasChangedOperation(ordersIndexDefinition));
            //endregion
        }
    }
}
